src/server/JasmineGraphServer.cpp

```diff
@@ -240,6 +241,25 @@ void JasmineGraphServer::start_workers() {
 
 }
 
+void JasmineGraphServer::waitForAcknowledgement(int numberOfWorkers) {
+    auto begin = chrono::high_resolution_clock::now();
+    int timeDifference = 0;
+    while (timeDifference < 30000) {
+        sleep(2);
```

Comment:
Here we sleep this loop for only 2ms which is quite low value. As a result the metadb gets queried very frequently which may stress the metadb. Can we increase this value to something like 50ms or 100ms if possible?

Reply:
The sleep function argument is in seconds. So it checks once every two seconds

Reply:
Ok